install/update_94_95.php

```diff
@@ -452,6 +452,37 @@ function update94to95() {
    $migration->addKey('glpi_tickettemplatepredefinedfields', 'tickettemplates_id');
    /** /ITiL templates */
 
+   if (!$DB->fieldExists('glpi_entities', 'autopurge_delay')) {
+      $migration->addField("glpi_entities", "autopurge_delay", "integer", [
+            'after'  => "autoclose_delay",
+            'value'  => Entity::CONFIG_NEVER
+         ]
+      );
+   }
+
+   CronTask::Register(
+      'Ticket',
+      'purgeticket',
+      7 * DAY_TIMESTAMP,
+      [
+         'mode'  => CronTask::MODE_EXTERNAL,
+         'state' => CronTask::STATE_DISABLE
```

Comment:
If it is disabled by default, admins that change the entity configuration may not understand why tickets are not purged. I think this cron should be active by default, even if it has nothing to do.

Reply:
I guess not, this may be destructive; admin have to enable such features explicitely.

Reply:
maybe a warning in the conf to alert the administrator the automatic is disabled (as the configuration and the action are separated) is this case, no ? Btw, i'm agree the action should be disabled by default

Reply:
Message added